arch/risc-v/src/esp32c3/esp32c3_i2c.c

```diff
@@ -799,8 +799,18 @@ static int esp32c3_i2c_sem_waitdone(struct esp32c3_i2c_priv_s *priv)
 
   clock_gettime(CLOCK_REALTIME, &abstime);
 
-  abstime.tv_sec += 10;
-  abstime.tv_nsec += 0;
+#if CONFIG_ESP32C3_I2CTIMEOSEC > 0
+  abstime.tv_sec += CONFIG_ESP32C3_I2CTIMEOSEC;
+#endif
+
+#if CONFIG_ESP32C3_I2CTIMEOMS > 0
+  abstime.tv_nsec += CONFIG_ESP32C3_I2CTIMEOMS * 1000 * 1000;
+  if (abstime.tv_nsec >= 1000 * 1000 * 1000)
+    {
+      abstime.tv_sec++;
+      abstime.tv_nsec -= 1000 * 1000 * 1000;
+    }
+#endif
```

Comment:
@gustavonihei one suggestion here:
Replace 1000 * 1000 by `NSEC_PER_MSEC`
and 1000 * 1000 * 1000 by `NSEC_PER_SEC`
See these macros at `nuttx/include/nuttx/clock.h`

Reply:
Nice suggestion, fixed! Thanks!